<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Datos de Clientes</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/persona.png</normaloff>../img/persona.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 2px solid #A5A5A5;
	border-radius: 10px;
}
QPushButton:hover {
	background-color: #b8cbff;

}

QLabel {
	text-align: right;
}

QLineEdit {
	border: 1px solid #CFCFCF;
	background-color: #FFFFFF;
	border-radius: 5px;
	
}
</string>
  </property>
  <widget class="QLineEdit" name="email">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>aceptar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/Ugrabar.ico</normaloff>../img/Ugrabar.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="ingrese un nombre"/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string comment="nombre" extracomment="nombre"/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string comment="nombre" extracomment="nombre"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="apellido">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="txt_nombre">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;NOMBRE :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_apellido">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;APELLIDO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_telefono">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;TELEFONO :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_email">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;E-MAIL :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefono">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="imagen">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/persona.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>cancelar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/Ucancelar.ico</normaloff>../img/Ucancelar.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>editar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/Ueditar.ico</normaloff>../img/Ueditar.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>agregar</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/Unuevo.ico</normaloff>../img/Unuevo.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>buscar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../img/Ubuscar.ico</normaloff>../img/Ubuscar.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
